### DISCUSSION TITLE
Galaxy Brain Achievement

### DISCUSSION THREAD

**Discussion by scarabaeus** [Schweinepriester/github-profile-achievements#313] [Q&A] [answered]
How are you doing?

**Comment by jokefrelon** [comment, marked as answer]
halo! i'm here

**Reply by scarabaeus** [reply]
@jokefrelon  We've got to do one more each to get it -- would you like to do another?

**Reply by jokefrelon** [reply]
Yes! Let's try it! I don't know if it's okay to answer the same person twice

**Reply by scarabaeus** [reply]
Haha I don't know either -- worth a shot.

**Reply by scarabaeus** [reply]
@jokefrelon https://github.com/Schweinepriester/github-profile-achievements/discussions/314

**Reply by jokefrelon** [reply]
@scarabaeus #315

**Reply by jokefrelon** [reply]
yes! i got the Galaxy Brain Achievement
